config/mtl.toml

```diff
@@ -55,6 +55,7 @@ base_offset = "0x2000"
 [fw_desc.header]
 name = "ADSPFW"
 load_offset = "0x40000"
+fw_image_flags = "0x12"
```

Comment:
Can you add a comment that `0x12` is `BASE_FW`+`CONTEXT_IS_NOT_SAVED` or something like this?

Reply:
thanks, updated